### DISCUSSION TITLE
What is the meaning of "inaccurate locations"?

### DISCUSSION THREAD

**Discussion by SiggiKeller** [owntracks/android#1837] [Q&A] [answered]
In the extended settings of the Android Client there is a setting "inaccurate locations". What is the meaning of this parameter? How does it work, how can I use it? 
I tried to find an explanation in the documentation and by searching the web, but it didn't become clear to me.

Thanks,
Siggi

**Comment by growse** [comment]
This one?

![image](https://github.com/user-attachments/assets/a715b925-67e9-4aff-818d-8796f1211ea1)

**Reply by SiggiKeller** [reply]
Exactly!

**Reply by growse** [reply]
It's the value which if a device-supplied location has an accuracy greater than it, then it's ignored.

**Reply by woj-tek** [reply]
> It's the value which if a device-supplied location has an accuracy greater than it, then it's ignored.

I'm just playing with owntracks and not getting timely updates (unless I open the app, which defies the point) so I was wondering about this config paramter but... above explication doesn't make sense to me. What does it mean "location acuracy"? Is it in meteres? 

That is "device reports accuracy at the best to 100m" so if I set it to 50 it will ignore such reported location?

And 0 ignores all inaccurate locations or disables the feature?

**Reply by ckrey** [reply]
The docs are pretty clear [booklet](https://owntracks.org/booklet/tech/json/#_typeconfiguration)
![IMG_1888](https://github.com/user-attachments/assets/2e40a695-c0b3-4b34-8488-d4fb26bee87b)

**Reply by woj-tek** [reply, marked as answer]
Yeah, this answers it. I was looking in android/user facing part of the book and search for "ignore inaccurate locations" didn't yield this bit of booklet.
